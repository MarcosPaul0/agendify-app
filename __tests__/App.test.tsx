// import App from '@app/index';
// import renderer from 'react-test-renderer';

describe('<Text />', () => {
  it('renders correctly', () => {
    // const tree = renderer.create(<App />).toJSON();
    // expect(tree).toBeDefined();
  });
});
