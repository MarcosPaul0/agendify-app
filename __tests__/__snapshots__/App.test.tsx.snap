// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Text /> renders correctly 1`] = `
<View
  style={
    [
      {
        "flexBasis": "0%",
        "flexGrow": 1,
        "flexShrink": 1,
      },
      {
        "alignItems": "center",
      },
      {
        "justifyContent": "center",
      },
      {
        "backgroundColor": "#22c55e",
      },
    ]
  }
>
  <Text
    style={
      [
        {
          "color": "#fff",
        },
      ]
    }
  >
    TESTE
  </Text>
</View>
`;
